tests/query/author.php

```diff
@@ -7,7 +7,7 @@
  * @group author
  */
 class Tests_Query_Author extends WP_UnitTestCase {
-	function setUp() {
+	function setup(): void {
```

Comment:
```suggestion
	public function setup(): void {
```

Reply:
I addressed this, but kept the capitalization.